knowledge-base/grid-select-or-deselect-item-on-row-click.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Select or Deselect Items on Row Click in Grid
+description: How to Select or Deselect Items on Row Click in Grid?
+type: how-to
+page_title: Select or Deselect Items on Row Click in Grid
+slug: grid-kb-select-or-deselect-item-on-row-click
+position: 
+tags: telerik, blazor, grid, select, deselect, row, click
+ticketid: 1534470
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+How can I select and deselect multiple rows by clicking on a row in a Grid with enabled selection?
+
+## Solution
+
+When selection is enabled in the Grid component, the built-in option for deselecting a row or selecting multiple rows is `Ctrl` + click. To deselect a row or select multiple rows by row clicking and without holding the `Ctrl` key, try the following:
```

Comment:
Remove `try`, we are giving solutions, and we should be aware if they are working

Reply:
Will do